library/HelloSign/AbstractResource.php

```diff
@@ -252,6 +267,26 @@ public function addFileUrl($file_url) {
         return $this;
     }
 
+    /**
+       * @param  boolean $use_preexisting_fields Set to true to use preexisting fields.
```

Comment:
Whoops, there are two spaces between `@param` and `boolean`

```
@param  boolean
      ^^
```

Reply:
I notice that this is done quite a bit, and I was beginning to think it was intentional, but I checked the [PHPDoc documentation for the `@param` tag](https://docs.phpdoc.org/references/phpdoc/tags/param.html) and they definitely don't use two spaces. Is using two spaces intentional? If so, why?

Reply:
Ah, my guess is that it was historically done so that the descriptions would be inline with the descriptions of `@return`s and `@throw`s?

```php
/**
 * @param  Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
 * @return Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
 * @throws Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
 */
```

I've never been a big fan of trying to align things in code, it always leads to a lot of unnecessary upkeep and it just leads to ugly code. But if this is how it's done in this repo, then I guess I'm not one to complain